locksmith/src/models/hookEvent.ts

```diff
@@ -1,7 +1,22 @@
 import { Table, Column, Model, AllowNull, DataType } from 'sequelize-typescript'
 
+interface HookEventAttributes {
+  id: number
+  network: number
+  lock?: string
+  key?: string
+  hookId: number
+  topic: string
+  body: any
+  state: string
+  lastError?: string
+  attempts: number
+  createdAt?: Date
+  updatedAt?: Date
```

Comment:
We have not added them to _all_. I guess it's only needed if we access these?

Reply:
it's not really needed. Nice to have for stricter typing. m00n added it so I'm keeping it because I'm going to remove the typescript package later.